packages/bot/src/stores/trade-animation-store.js

```diff
@@ -0,0 +1,40 @@
+import { observable, action, reaction } from 'mobx';
+
+const CONTRACT_STATUS = [
+    'Bot is not running',
+    'Attempting to Buy',
+    'Buy Succeeded',
+    'Bot is stopping',
+    // 'Closing Contract',
+    'Contract Closed',
+];
+
+export default class TradeAnimationStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+    }
+
+    @observable status_class = ['', '', ''];
+    @observable status_title = CONTRACT_STATUS[0];
+
+    @action.bound onMount () {
```

Comment:
This function isn't modifying any observable, but the callback passed to the `reaction` does, so that callback should be marked as an action instead.

Reply:
for some reason, `status_class` did not update if I removed `.bound`

Reply:
`this.root_store.flyout.contract_status` can't be access if `action.bound` is not bind to `onMount` function